be/src/io/cache_input_stream.cpp

```diff
@@ -215,30 +266,87 @@ void CacheInputStream::_deduplicate_shared_buffer(SharedBufferedInputStream::Sha
     sb->size = end - sb->offset;
 }
 
+struct ReadFromRemoteIORange {
+    ReadFromRemoteIORange(const int64_t offset, char* write_pointer, const int64_t size)
+            : offset(offset), write_pointer(write_pointer), size(size) {}
+    const int64_t offset;
+    char* write_pointer;
+    const int64_t size;
+};
+
 Status CacheInputStream::read_at_fully(int64_t offset, void* out, int64_t count) {
-    BlockCache* cache = BlockCache::instance();
+    const int64_t origin_offset = offset;
```

Comment:
could you consider to use `const int64_t` here?

Reply:
It's already use const `int64_t` here  🤣